features/in_onderzoek.feature

```diff
@@ -346,574 +414,708 @@ Functionaliteit: in onderzoek
+
   Rule: wanneer een groep in de bron in onderzoek is, wordt elk gegeven in het antwoord dat gevuld wordt uit een van de elementen in die groep ook in inOnderzoek opgenomen met de waarde true
     - de elementcode voor onderzoek eindigt op 00
 
     Scenario: persoon heeft hele groep naam in onderzoek
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                        | waarde    |
-      | burgerservicenummer         | 555550001 |
-      En de persoon heeft de volgende 'persoonInOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | 010200   |
-      | datumIngangOnderzoek          | 20220307 |
+      Gegeven de persoon met burgerservicenummer '000000371' heeft de volgende gegevens
+      | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | 
+      | 010200                          | 20220307                       |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                                                                                                              |
       | type                | RaadpleegMetBurgerservicenummer                                                                                     |
-      | burgerservicenummer | 555550001                                                                                                           |
+      | burgerservicenummer | 000000371                                                                                                           |
       | fields              | naam.voornamen,naam.adellijkeTitelPredicaat,naam.voorvoegsel,naam.geslachtsnaam,naam.aanduidingNaamgebruik,geboorte |
-      Dan heeft de persoon met burgerservicenummer '555550001' alleen de volgende 'naam.inOnderzoek' gegevens
-      | naam                             | waarde     |
-      | voornamen                        | true       |
-      | adellijkeTitelPredicaat          | true       |
-      | voorvoegsel                      | true       |
-      | geslachtsnaam                    | true       |
-      | datumIngangOnderzoek.datum       | 2022-03-07 |
-      | datumIngangOnderzoek.type        | Datum      |
-      | datumIngangOnderzoek.langFormaat | 7 maart 2022 |
-      En heeft de persoon GEEN 'inOnderzoek' gegevens
-      En heeft de persoon GEEN 'geboorte.inOnderzoek' gegevens
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                                              | waarde       |
+      | naam.inOnderzoek.voornamen                        | true         | 
+      | naam.inOnderzoek.adellijkeTitelPredicaat          | true         |
+      | naam.inOnderzoek.voorvoegsel                      | true         |
+      | naam.inOnderzoek.geslachtsnaam                    | true         |
+      | naam.inOnderzoek.voorletters                      | true         |
```

Comment:
voorletters en volledigeNaam worden niet gevraagd. Dan zou hun inOnderzoek property niet mogen voorkomen in de response

Reply:
Klopt is aangepast.